plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -678,6 +683,28 @@ nothrow:
         }
     }
 
+    void opCall(ref IncreaseFilePrio data) {
+        try {
+            spinSql!(() @trusted {
+                string fileContent = readText(conf.metadataPath);
```

Comment:
start the method by checking if the file exist.
if it is empty, do nothing.
if metadataPath is set but the file do not exist print a warning (logger.warning) and early return.
if the file fails to be read then print that as a specific message.
if parseJSON fails to parse the content print that as a specific message.
etc

consider the user. if it fails then the user need to know what went wrong and how to fix it.